configure.ac

```diff
@@ -221,6 +221,19 @@ then
 fi
 
 
+AC_ARG_ENABLE([norng],
+    [  --enable-norng       Enable Test Cert (default: disabled)],
```

Comment:
Perhaps the ./configure can be --disable-rng? Then default is no def and on disable define WC_NO_RNG?